source/panel_predictor/panel_finder/panel_classifier/__main__.py

```diff
@@ -1,4 +1,4 @@
-from .panel_classifier import PanelClassifier
+from panel_classifier import PanelClassifier
```

Comment:
I think the relative import is important here, as `panel_classifier` isn't in the root directory

Reply:
does this work for you? I received an error on this

Reply:
Right, forgot about the Pycharm folks

Reply:
Keep in mind that it's not pycharm that would make this work - it's the way your python environment is set up.